sysroot/System/Configs/man-pages.json

```diff
@@ -0,0 +1,43 @@
+{
+    "echo": {
+    	"description": "prints text to the terminal",
+    	"usage": "echo [STRING]",
+    	"example": "echo hello world: hello world"
+    },
+    
+    "uptime": {
+    	"description": "returns how long the computer has been on for",
+    	"usage": "uptime",
+    	"example": "uptime: Up 8 minutes, 9 seconds"
+    },
+    
+    "man": {
+    	"description": "man is short for manual. It provides documentation on commands",
+    	"usage": "man [OPTIONS]",
+    	"example": "man echo uptime"
+    },
+    
+    "sysfetch": {
+    	"description": "grabs system information such as the kernel, version, uptime, shell, etc",
+    	"usage": "sysfetch",
+    	"example": "sysfetch"
+    }
```

Comment:
This JSON file is malformed, you have missings `,`

Reply:
thanks for finding that, when I tested it, it ran fine. This is seems to be a bug in the JSON reader I think

Reply:
oops, I forgot to give a commit name. I wish github allowed you to rename your commits

Reply:
yes, you can if you git push --force

Reply:
ok, I changed the name by it making a new commit

Reply:
I have followed your suggestion about the structure: https://github.com/skiftOS/skift/pull/84#issuecomment-667859390